conf/config-default.yaml

```diff
@@ -470,3 +470,4 @@ plugin_attr:
       connect: 60s
       read: 60s
       send: 60s
+  redirect_https_port: 8443   # the default port for use by HTTP redirects to HTTPS
```

Comment:
We should use a structure like this:
```
redirect:
    https_port: ...
```
and comment it out as nobody wants to be redirected to 8443 by default.

Reply:
done